packages/amplication-server/src/graphql/graphqlSubscriptionPubSubKafka.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable, OnModuleInit } from "@nestjs/common";
+import { Kafka, KafkaConfig } from "kafkajs";
+import { KafkaPubSub } from "graphql-kafkajs-subscriptions";
+import { createNestjsKafkaConfig } from "@amplication/util/nestjs/kafka";
+import { KAFKA_TOPICS } from "@amplication/schema-registry";
+
+@Injectable()
+export class GraphqlSubscriptionPubSubKafkaService implements OnModuleInit {
+  private pubsub: KafkaPubSub;
+
+  async onModuleInit() {
+    const config = createNestjsKafkaConfig().options;
+
+    const kafka = new Kafka(config.client as KafkaConfig);
+
+    this.pubsub = await KafkaPubSub.create({
+      kafka,
+      topic: KAFKA_TOPICS.SHARED_GRAPHQL_SUBSCRIPTION_PUBSUB_TOPIC,
+      groupIdPrefix: config.consumer.groupId + "-graphql-subscrition",
```

Comment:
Is this groups ID constant for all server instances? something like "amplication-server-group-prod" ?

Reply:
yep, but currently that is a prefix as the graphql-kafkajs-subscriptions lib doesn't support custom fixed groupids. 
i.e. if we have 2 runtime of server running we will see 3 consumer groups for server like:
<img width="545" alt="image" src="https://github.com/amplication/amplication/assets/2861984/063a26c0-6484-4a8e-b308-b4483afb4c46">